### ISSUE TITLE
Fields are not nullable by default

### ISSUE THREAD

**Issue by DontPanicO** [collerek/ormar#339]
**Description**
Fileds nullable attribute is always set to false even if the doc states:
```c
nullable: bool = not primary_key -> defaults to False for primary key column, and True for all other.
```
See [https://collerek.github.io/ormar/fields/common-parameters/](https://collerek.github.io/ormar/fields/common-parameters/) 

This is in apparently in contrast with the method `is_filed_nullable` in ormar/fields/model_fieldspy:

```python
if nullable is None:
    return (
        default is not None
        or server_default is not None
        or (pydantic_only is not None and pydantic_only)
    )
return nullable
```
Since ModelFieldFactory initialize the nullable value as None If not specified (line 77, same file), the if statement is triggered.
There, the return condition is always returning False unless at least one of specified attributes is True (or better, is not None). So having a simple filed defined as:
```python
name: str = ormar.String(max_length=255)
```
will cause the nullable attr to be set as False.

I've never looked at the source code before today, so forgive me if I'm missing something here.

**To Reproduce**
1. Create a simple user model defining fields without declaring attributes that impact the `is_nullable_field` behavior. E.g.
```python
class User(ormar.Model):
    class Meta:
        # your database and metadata

    id: int = ormar.Integer(primary_key=True, autoincrement=True, index=True)
    created: datetime.datetime = ormar.DateTime(timezone=True, server_default=func.now())
    username: str = ormar.String(max_length=72, nullable=False)
    email: EmailStr = ormar.String(max_length=120, nullable=False)
    password: str = ormar.String(max_length=255, nullable=False)
    is_active: str = ormar.Boolean(default=True)
    is_superuser: str = ormar.Boolean(default=False)
    first_name: str = ormar.String(max_length=56)
    last_name: str = ormar.String(max_length=56)
```
2. Try to create an instance of that model without first_name and last_name arguments
```python
u1 = User(username='someuser', email='some@user.com', password='helloworld')
```
3. Raise the exception

**Expected behavior**
Non primary key fields should be nullable by default, or the documentation should be updated.

**Screenshots**
Unfortunately I can not provide a screen right now. I'm posting it tomorrow morning (8:00 am Amsterdam Time)

**Versions**
 - Database backend used (sqlite)
 - Python version 3.9.6
 - `ormar` version 0.10.18
 - `pydantic` version 1.8.2

**Comment by DontPanicO** [comment]
I post two screens here.

![ormar_nullable_fields](https://user-images.githubusercontent.com/72745847/132456343-4cd205c7-32f1-47bd-9a0c-3b15888eae0b.png)

![ormar_nullable_field_with_optional](https://user-images.githubusercontent.com/72745847/132456378-3ad1d5ea-ef2d-4cb6-bc0d-8e61c1b0dbac.png)

**Comment by collerek** [comment]
Yes, you are right, this was changed quite some time ago and by default, all fields are required (except relation fields, autoincrement pk, or fields with defaults). So it's intentional and the bug is in the documentation.

Thanks for catching that!

**Comment by DontPanicO** [comment]
Thanks to you , for building this awesome orm.

**Comment by collerek** [comment]
Fix the docs in #346
